pkg/injector/pod_patch.go

```diff
@@ -458,8 +461,12 @@ func isResourceDaprEnabled(annotations map[string]string) bool {
 	return getBoolAnnotationOrDefault(annotations, daprEnabledKey, false)
 }
 
-func getKubernetesDNS(name, namespace string) string {
-	return fmt.Sprintf("%s.%s.svc.cluster.local", name, namespace)
+func getServiceAddress(name, namespace, clusterDomain string, port int) string {
+	return fmt.Sprintf("%s:%d", getKubernetesDNS(name, namespace, clusterDomain), port)
```

Comment:
Please, do it all in a single Sprintf statement.

Reply:
Ok, i will adjust it later.